build.install4j

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <install4j version="7.0.1" transformSequenceNumber="7">
```

Comment:
@ssoloff Also we could consider updating install4j to 7.0.3 and the bundled jre to u152 for "important bugfixes and security related patches": http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
Or we wait unti java 9 is available at java.com and bundle a java 9 JRE with TripleA then

Reply:
@RoiEXLab The one issue I have with adding another "active" JRE to the triplea-game/assets repo (i.e. one we have to keep on `HEAD` to support the installer in older releases) is the additional download time we will incur when building from Gradle (as well as the size increase of the repo for all time).

Can we use one of our Linode servers to act as a static file host for the JREs?  That way we can avoid adding those large files to the repo in the first place.  I'm assuming Linode charges for bandwidth, so this may not be the best plan if we have hundreds or thousands of people downloading a ~350 MiB file each month.

Reply:
We used up 24 out of 10000GB bandwith this month so far, I don't think the fees are going to drastically increase because of this, the credentials are all in the secrets file, so you should be able to  login as root on one of the bot servers, just tell @prastle not to shut them down when there is low lobby activity

Reply:
Cool!  I'll open a new issue to discuss this.